src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,42 @@
+.calendar {
+  width: 7 * ($day-size + $gap) - $gap + 2 * $calendar-padding;
```

Comment:
The formula used to calculate the width of the calendar should include parentheses to ensure the correct order of operations. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), arithmetic operations should be enclosed in parentheses for clarity. It should be: width: (7 * ($day-size + $gap) - $gap) + (2 * $calendar-padding);